webapp/app/[locale]/_components/navbar/index.tsx

```diff
@@ -23,6 +23,7 @@ import { HemiLogoFull } from './_components/hemiLogo'
 import { ItemLink, ItemWithSubmenu, NetworkSwitch } from './_components/navItem'
 import { SocialLinks } from './_components/socialLinks'
 import { TermsAndConditions } from './_components/termsAndConditions'
+import { CmcAttribution } from './_components/cmcAttribution'
```

Comment:
ESLint warning to fix here.

Reply:
why prettier doesn't get that?

Reply:
I think I know why... I'll send a different PR to fix it

Reply:
fixed